### PR TITLE
feat: support helm deployment

### PR DESCRIPTION
**Pull request by matrixji**
issue: #1991

Signed-off-by: Ji Bin <matrixji@live.com>

### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation
- [x] This PR is using a `label` (bug, feature etc.)
- [x] My code is has necessary documentation (if appropriate)
- [ ] I have added any relevant tests
- [x] This section (**⚠️ &nbsp;&nbsp;Pre Checklist**) will be removed when submitting PR

# Summary

### Does this close any open issues?
Partly #1991 

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: deployment/helm/templates/deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          ports:
+            - containerPort: 80
+          envFrom:
+            - configMapRef:
+                name: {{ include "devlake.fullname" . }}-config
+          env:
+            - name: DEVLAKE_ENDPOINT
+              # TODO: remove hardcoded `cluster.local`
+              value: {{ include "devlake.fullname" . }}-lake.{{ .Release.Namespace }}.svc.cluster.local:8080
+            - name: GRAFANA_ENDPOINT
```
Comment (klesh) [inline deployment/helm/templates/deployments.yaml:77]: This should be the public URL of Grafana, which will be sent directly to the user's browser, for the user to click and open up grafana dashboard.

File: deployment/helm/templates/statefulsets.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            timeoutSeconds: 30
+          envFrom:
+            - configMapRef:
+                name: {{ include "devlake.fullname" . }}-config
+          env:
+            - name: DB_URL
+              value: mysql://{{ .Values.mysql.username }}:{{ .Values.mysql.password }}@{{ include "devlake.fullname" . }}-mysql:3306/{{ .Values.mysql.database }}?charset=utf8mb4&parseTime=True
+          command: ["/bin/sh"]
+          # workaround for keep .env in in pv
+          args: ["-c", "touch /app/config/.env; ln -s /app/config/.env /app/.env; lake"]
```
Comment (klesh) [inline deployment/helm/templates/statefulsets.yaml:114]: Neat! 
But we had added `ENV_PATH` environment variable for specifying the location of  `.env` file, would you kindly consider using this variable?

**Review by klesh — APPROVED** [review]
This is a great implementation, very professional! I like it very much.
Sorry for reviewing it so later, I would like it to be merged ASAP.
If @matrixji could find time to make some minor improvements, or we could merge it today, and improve them later

File: deployment/helm/templates/deployments.yaml (hunk 1 above)
Comment (matrixji) [reply deployment/helm/templates/deployments.yaml:77]: Yes, fixed. Currently introducing a parameter for it. And later I add ingress support and will overwrite it with ingress's endpoint when ingress is configured.

File: deployment/helm/templates/statefulsets.yaml (hunk 2 above)
Comment (matrixji) [reply deployment/helm/templates/statefulsets.yaml]: Now with ENV_PATH=/app/config/.env, That's smarter, thanks for your advice.

**Comment by matrixji** [comment]
Comments fixed and rebased from upstream.

**Pull request merged**